### DISCUSSION TITLE
KubeCon EU 2023 Co-located Event

### DISCUSSION THREAD

**Discussion by christianh814** [open-gitops/project#142] [Polls]
The CNCF is going to provide us a room for 1/2 a day at KubeCon EU in Amsterdam. We can use this time however we would like and this discussion is to vote and/or give ideas.

**Comment by christianh814** [comment]
I am a fan of "Open Forum". Let's discuss what we're doing in prod and get feedback!

**Reply by niklasmtj** [reply]
Thats a great idea! Maybe we can also think about in advance how we would structure that and pick some topics in advance but also some when we're starting on that day. So this way we can get everyone involved even when they're not active here yet.

**Reply by makkes** [reply]
We have some on-going discussions here on GitHub. Maybe we could pick some up to discuss in-person with the goal of producing a tangible outcome?

Examples:

- https://github.com/open-gitops/project/discussions/96
- https://github.com/open-gitops/project/discussions/104
- ...

**Reply by scottrigby** [reply]
I like this @makkes. We can do this BoF style as well
